src/main/java/com/techzen/techlearn/dto/request/ChapterRequestDTO.java

```diff
@@ -13,5 +14,11 @@ public class ChapterRequestDTO {
     @NotBlank(message = "Name is required")
     private String name;
 
+    @NotNull(message = "Order is required")
```

Comment:
dùng @FieldDefaults(level = AccessLevel.PRIVATE) cho gọn code

Reply:
Các thông báo phải để trong ErrorCode chứ không phải thêm ở đây
vd như @NotBlank(message = "LESSON_TITLE_INVALID")
    String title;
    LESSON_TITLE_INVALID(180002, "Lesson title must be not blank", HttpStatus.NOT_FOUND),